lib/dry/inflector.rb

```diff
@@ -268,6 +268,18 @@ def uncountable?(input)
       !(input =~ /\A[[:space:]]*\z/).nil? || inflections.uncountables.include?(input.downcase)
     end
 
+    def check_for_acronyms(word)
+      acronyms?(word) ? acronyms(word) : word.capitalize
```

Comment:
Could scrap the `acronyms?` and `acronyms` methods and just call `inflections.acronyms.fetch(input) { input.capitalize }` here

Reply:
Woow Fantastic suggestion @AMHOL